server/methods/setLastVisitedRoom.js

```diff
@@ -0,0 +1,8 @@
+Meteor.methods({
+	setLastVisitedRoom(userId, room) {
```

Comment:
Don't pass in the userId to this method. Instead, ensure that the user is logged in and then if they are user the logged in users' id. If you need to see how this is done, look around in the directory this file resides in and you'll see how to.